src/WebJobs.Script/Utility.cs

```diff
@@ -1028,7 +1028,8 @@ public static void ValidateRetryOptions(RetryOptions
         // WORKER_INDEXING_DISABLED contains the customers app name worker indexing is then disabled for that customer only
         public static bool CanWorkerIndex(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, FunctionsHostingConfigOptions functionsHostingConfigOptions)
         {
-            if (environment.IsLogicApp())
+            // NOTE: Enabling the worker indexing for Logic Apps with codeful mode enabled.
+            if (environment.IsLogicApp() && !environment.IsLogicAppCodefulModeEnabled())
```

Comment:
@apranaseth Lets sync up offline on this. Would love to know more about the end to end testing and scenarios supported by this flag.